Desktop/components/JASP/Widgets/FileMenu/FileList.qml

```diff
@@ -17,6 +18,29 @@ ListView
 
 	spacing:				10
 	model:					cppModel
+	keyNavigationWraps:		true
+
+	Keys.onEscapePressed:	resourceMenu.forceActiveFocus();
+	Keys.onTabPressed:		(event) =>
+							{
```

Comment:
Id suggest just setting "event.accepted=true" first and then it might be "unaccepted" in the else clause but that should work out fine. Now It'd be just false all the time right?

Reply:
event.accepted is set to true by default for the specific key handlers.
In contrast to general handlers like onPressed etc.